### DISCUSSION TITLE
Amiga / no refresh of diskname (diskinfo)

### DISCUSSION THREAD

**Discussion by Kanz1A** [keirf/flashfloppy#592] [Q&A] [answered]
Hi there,
ive got a problem with the refresh of the diskname.
Gotek: SFRC922D / Amiga 1200 extern or A2000 intern / Workbench 3.2.1
Jumper: only on S0
FF: 3.29

When i boot up the system is recognizing the adf in the Gotek and names the disk after the *.adf-file
for example, 
on bootup to WB 3.2.1
slot 1 = HistoryLine disk 1/7 = name in WB: HL1
when i then switch with the button to slot 2 = HistoryLine disk 2/7 
the name of the disk in WB stays : HL1 
when i am having a look into the folder, it contains the files of disk 2.

The problem is, that the game will not recognize disk 2 as disk 2 because the name stays disk1

at all.... i dont know why the name of the disk it not refreshing when changing the adf-slot

perhaps anyone can help me

ps: i used the drive some month ago and it worked fine

**Comment by Kanz1A** [comment]
The disk always keeps the diskname of the first adf that is loaded

**Comment by keirf** [comment, marked as answer]
Sounds like disk change signal (pin 2) not working properly. Either a hardware problem or perhaps `interface=ibmpc` or similar from an old FF.CFG has been cached in the Gotek?

**Reply by Kanz1A** [reply]
you mean pin 2 of the IDE-Header?
.# Floppy-drive interface mode
.# shugart: P2=DSKCHG, P34=RDY
.# ibmpc: P2=unused, P34=DSKCHG
.# ibmpc-hdout: P2=HD_OUT, P34=DSKCHG (not generally needed: prefer 'ibmpc')
.# akai-s950: P2=HD_OUT, P34=RDY (Akai S950)
.# amiga: P2=DSKCHG, P34=DRIVE_ID (not generally needed: prefer 'shugart')
.# jc: JC closed: ibmpc, JC open: shugart
interface = shugart

thats my FF.cfg

how can i cache this to the gotek?
isnt it changed to standart with every flash over ttl?

mh,... the files on the amiga are as well not the right one. musst be somewhat wrong in the cache.

**Reply by keirf** [reply]
Well the Amiga is not detecting disk changes but your FF.CFG setting looks ok. So perhaps it's a hardware issue.

**Reply by Kanz1A** [reply]
think the same... hardware issue.....

**Reply by Kanz1A** [reply]
> Well the Amiga is not detecting disk changes but your FF.CFG setting looks ok. So perhaps it's a hardware issue.

is there a way for a hard reset? to write the cache new? or is this by rewriting via TTL?

**Comment by keirf** [comment]
Reflashing via TTL will do it. Also so will holding both buttons for a few seconds *after* power on, with no stick inserted, until display changes.

**Comment by Kanz1A** [comment]
mh tried both again, but still not working.

When i try my other goteks, the floppydrive-icon disappears on WB  for a second when i change the slot number, and than it loads the floppy new.

On this "broken" one, the floppy-icon stays, it does not change the name and only shows "40" for a short time on the display.
When i change the slot number an reboot the system, it loads the adf that is behind the number i have choosen before and the disk works fine.
Must be an error on reloading.

think its like you said.... its a hardware error that does the drive not get the signal to eject the old und load the new file.

thanks for trying to help me
